src/components/structures/RoomView.js

```diff
@@ -349,14 +371,14 @@ module.exports = React.createClass({
             // if the event after the one referenced in the read receipt if sent by us, do nothing since
```

Comment:
well, I could... but I'd kinda prefer to keep that out of this branch as it's not really related and I'd prefer to avoid the risk of conflicts if possible.